coalib/settings/ConfigurationGathering.py

```diff
@@ -185,6 +186,21 @@ def warn_nonexistent_targets(targets, sections, log_printer=None):
         raise SystemExit(2)  # Invalid CLI options provided
 
 
+def warn_bears_absent(bears, log_printer=None):
+    """
+    Checks if coala-bears was also installed.
+
+    :param bears:        A dictionary with bears as keys and list of
+                         sections containing those bears as values.
+    :param log_printer:  A log printer to emit the warning to.
+    """
+    if not bears:
+        logging.error('No bears are installed. Did you forget to install '
+                      'the `coala-bears` package? Try `pip3 install '
+                      'coala-bears`.')
+        raise SystemExit(2)
```

Comment:
CI build failed due to the failing tests, you should modify the tests to fix it.
I think the `SystemExit` call caused the tests failing, and you can fix it like this:
```
with self.assertRaises(SystemExit) as cm:
    your test code
```